core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -50,6 +57,37 @@ object ZkUtils extends Logging {
   val EntityConfigPath = "/config"
   val EntityConfigChangesPath = "/config/changes"
 
+  /** true if java.security.auth.login.config is set to some jaas file which has "Client" entry. **/
+
+  val isSecure: Boolean = {
+    val loginConfigurationFile: String = System.getProperty("java.security.auth.login.config")
```

Comment:
I know that this is how JAAS is normally configured, but we don't usually use system properties to configure Kafka. Should we be providing an alternative way?